optuna/storages/_redis.py

```diff
@@ -618,20 +649,31 @@ def get_all_trials(
         states: Optional[Tuple[TrialState, ...]] = None,
     ) -> List[FrozenTrial]:
 
+        trials = self._get_trials(study_id, states, set())
+
+        if deepcopy:
+            return copy.deepcopy(trials)
+        else:
+            return trials
+
+    def _get_trials(
```

Comment:
Could you add unit tests for this method? esp, for exlucded_trial_ids.

Reply:
Ok, fixed. I reused the `test_get_trials_excluded_trial_ids()` of `RDBStorage` test.